spec/controllers/state_file/questions/nj_disabled_exemption_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+
+RSpec.describe StateFile::Questions::NjDisabledExemptionController do
+  let(:intake) { create :state_file_nj_intake }
+  before do
+    sign_in intake
+  end
+
+  describe "#edit" do
+    render_views
+    it 'succeeds' do
+      get :edit
+      expect(response).to be_successful
+    end
+  end
+
+  describe "#show" do
+    context "when both taxpayer and spouse claimed DirectFile blind" do
+      let(:intake) { create :state_file_nj_intake, :married_filing_jointly, :primary_blind, :spouse_blind }
+      it "does not show" do
+        expect(described_class.show?(intake)).to eq false
+      end
+    end
+
+    context "when spouse or taxpayer have not claimed DirectFile blind" do
```

Comment:
[dust] I'm confused by the "or" + intake not including a blind primary or spouse. To me this test name makes me think we're testing that it shows when one of them has claimed the blindness exemption and the other has not. So I would want to change it to "and" or add either :primary_blind or :spouse_blind

Reply:
This was originally meant to be `when both spouse and taxpayer have not claimed DirectFile blind`. Added a different test for `when taxpayer has claimed DirectFile blind but spouse has not` in the next commit.